README.develop.md

```diff
@@ -0,0 +1,52 @@
+# Development Guide
```

Comment:
The standard filename we are using for this is CONTRIBUTING.md

Reply:
Looks like `CONTRIBUTING.md` already exists (last edit 4 years ago!). I will starting rationalizing and merging the two.

Reply:
Merged.